### DISCUSSION TITLE
API key pending approval  and it's day 13

### DISCUSSION THREAD

**Discussion by Wuzoobia** [etsy/open-api#1312] [Q&A]
Hello everyone,
I apologize if I'm posting this in the wrong place, but I'm in need of some guidance regarding the API key application process. 
It's been almost two weeks since I submitted my request for an API key, and I noticed that it is still pending approval. I wanted to find out from this Github discussion if there's a common timeframe for approval. For context, I currently operate an active Etsy store, and I’m eager to integrate the API to enhance my shop's functionality.
Any insights you could provide about the typical wait time or the approval process would be greatly appreciated.

**Comment by Wilderbill** [comment]
You just have to be patient.
It took several weeks for me to be approved.
That is on top pf my first several-week attempt which was denied with "Missing valid website." ???

**Comment by bindalf** [comment]
did you get the buyer_email field?

**Comment by Wilderbill** [comment]
I do not know what the "buyer_email" field is. The closest thing might be the "email_r" Etsy API scope. Anyway, at this point I have it all working. I have my scopes set to "listings_r listings_w listings_d email_r", but so far have not deleted or emailed through the API. Just reading, altering and the writing.
